src/styles/card.scss

```diff
@@ -0,0 +1,110 @@
+.card {
+  width: 200px;
+  padding: 15px 14px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  font-family: "Roboto", sans-serif;
+
```

Comment:
**Missing Hover Effects**:

1. Hover effects should be smooth over 300ms
2. Increase the card size by 20% on hover without affecting neighboring cards
3. Change the card title text color to _**#34568b**_ on card hover